src/dmd/tokens.d

```diff
@@ -81,7 +81,7 @@ enum TOK : int
     delegatePointer,
     delegateFunctionPointer,
 
-    // 54
+    // 54 // TODO: instead of these non robust numbers, use `cast(TOK) number`
```

Comment:
The numbers are there because when I print out a TOK value, in order to figure out which one it is, I have to count the tokens.